PD-290: Create PVCs with ReadOnlyMany access mode (#574)
portworx/pxdocs: content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-readonlymany-pvcs.md

---
title: Create PVCs using the ReadOnlyMany access mode
weight: 12
keywords: Portworx, container, Kubernetes, storage, k8s, PV, persistent disk, PVC, ReadOnlyMany
description: Learn how to create PVCs using the ReadOnlyMany access mode
series: k8s-vol
---

> This guide shows how to mount a disk in `ReadOnlyMany`(ROM) access mode. The in-tree Portworx driver for Kubernetes does not support creating PVCs with `ReadOnlyMany(ROM)` access mode. To achieve this functionality, follow the below steps:

Reviewer: ```suggestion
This guide provides steps for mounting a disk in the `ReadOnlyMany`(ROM) access mode. The in-tree Portworx driver for Kubernetes does not support creating PVCs with the `ReadOnlyMany(ROM)` access mode. To achieve this functionality, follow the steps below:
```

Reviewer 2: Remove  "a disk".  instead how about, This guide provides steps for mounting a portworx volume in `ReadOnlyMany` ?

1. Create a `sharedv4` volume. Note that you can access a `sharedv4` volume from multiple pods at the same time. For details about creating a `sharedv4` volume, refer to the [Create sharedv4 PVCs](/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-sharedv4-pvcs/) page. 

> 2. Include a `persistentVolumeClaim` in the `volumes` section of your Pod and set the `readOnly` parameter to `true`.

Reviewer: ```suggestion
2. Include a `persistentVolumeClaim` in the `volumes` section of your Pod and set the `readOnly` parameter to `true`:
```

> 2. Include a `persistentVolumeClaim` in the `volumes` section of your Pod and set the `readOnly` parameter to `true`.

Reviewer: Do they link the SharedV4 PVC to the pod here?

>     Example:

Reviewer: ```suggestion

```

>     Example:

Reviewer: ```suggestion
```

>         ```text

Reviewer: This indentation doesn't appear to render correctly on my local build.

Author: @nathan-px Fixed.
